### ISSUE TITLE
app crashes on opening preferences in API 19

### ISSUE THREAD

**Issue by jaztriumph** [openfoodfacts/openfoodfacts-androidapp#1153]
**Checklist**
- [x] Bug Report - The issue you are reporting might be there already. Please search on the [issues tracker](https://github.com/openfoodfacts/openfoodfacts-androidapp/issues) before creating or reporting an issue.
- [x] Before reporting, please ensure (if you can) that you have tested it with the latest (dev) version of the software.
- [x] Don't forget to tag your issue, it makes it easier to discover.
**Summary:** 

App crashes on clicking preferences in lower sdk versions.

**Add system logs:**

```java.lang.NullPointerException
at openfoodfacts.github.scrachx.openfood.fragments.PreferencesFragment.onCreatePreferences(PreferencesFragment.java:73)
at android.support.v7.preference.PreferenceFragmentCompat.onCreate(PreferenceFragmentCompat.java:224)
at android.support.v4.app.Fragment.performCreate(Fragment.java:2246)
at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1377)
at android.support.v4.app.FragmentTransition.addToFirstInLastOut(FragmentTransition.java:1187)
at android.support.v4.app.FragmentTransition.calculateFragments(FragmentTransition.java:1070)
at android.support.v4.app.FragmentTransition.startTransitions(FragmentTransition.java:115)
at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2374)
at android.support.v4.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2332)
at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2239)
at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:700)
at android.os.Handler.handleCallback(Handler.java:808)
at android.os.Handler.dispatchMessage(Handler.java:103)
at android.os.Looper.loop(Looper.java:193)
at android.app.ActivityThread.main(ActivityThread.java:5333)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:515)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:829)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:645)
at dalvik.system.NativeStart.main(Native Method))
```

**Expected behavior:** 

preferences should be opened

**Device and Android version:** 

HTC 820 g+, KIT-KAT,api 19.

**Screenshots:** 

![preference](https://user-images.githubusercontent.com/22943390/37097577-cf997f6a-2241-11e8-8ace-5d4290ee4ec5.gif)

**Comment by l-o-b-s-t-e-r** [comment]
Probably fixed here: #1115 
I added nullable reference check.

**Comment by jaztriumph** [comment]
@l-o-b-s-t-e-r I will close this after that PR is merged.
